cms/core/models/snippets.py

```diff
@@ -53,3 +58,61 @@ def save(self, *args: Any, **kwargs: Any) -> None:
 
     def __str__(self) -> str:
         return str(self.name)
+
+
+class GlossaryTerm(TranslatableMixin, PreviewableMixin, RevisionMixin, index.Indexed, models.Model):
+    """A model for glossary terms."""
+
+    title = models.CharField(max_length=255)
```

Comment:
suggestion: `name` feels a lot more natural, now that I see this in practice.

Reply:
I was also thinking about `term`, but I guess that might pretty easily get confused with the snippet itself in different contexts.

Reply:
https://github.com/ONSdigital/dis-wagtail/pull/110/commits/b8bb7a7c6b099cac41b09b2e205ba7b25d7c240c and https://github.com/ONSdigital/dis-wagtail/pull/110/commits/79aebb441948c40a93cebd6c796b0c263cc52a5c